src/commands/subscription.ts

```diff
@@ -131,52 +168,73 @@ function formatNoTeamResponse() {
     return "You can't use my-team or my-team-channel since you don't have a team right now."
 }
 
+export function emitEvent(eventName: string, _league: League, sources: [string, string], context: Context): void {
```

Comment:
It might even just make sense to get rid of the emitter pattern entirely, only thing it’s really using now is the reference to the bot object